### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cymptomlabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by roysagi** [comment]
@cymptomlabs - Hi!
Thank you very much for contributing!
Please [sign the CLA](https://cla-assistant.io/demisto/content?pullRequest=12336) so we can continue.

**Comment by marcellarichmond** [comment]
@mchasepan

File: Packs/Cymptom/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### What does this pack do?
+
+By leveraging the integration between Cortex® XSOAR and Cymptom’s security posture management solution, security teams can automate and optimize complex workflows across the full stack of information systems and security tools. 
+Together, Cortex XSOAR and Cymptom’s integrated solutions provide always-on risk visibility that security teams need throughout their on-premise and cloud-based networks. 
+By leveraging this integration, your team optimizes existing security tools and workflows with full visibility of vulnerabilities and prioritized risk. 
+- Integrate always-on risk & vulnerability assessment with the rest of your security stack and workflows for more comprehensive visibility and a continuous risk remediation.
```
Comment (sangsoojun88) [inline Packs/Cymptom/README.md:6]: This sentence is a little confusing. Would be better if we can change it to:
"Integrate with Cymptom's always-on comprehensive risk & vulnerability assessment with the rest of your security stack and workflows for more comprehensive visibility and a continuous risk remediation.

**Comment by cymptomlabs** [comment]
Accepted the changes. @content-bot 
Thanks.

**Comment by sangsoojun88** [comment]
Everything looks good on my end

**Review by roysagi — APPROVED** [review]

**Pull request merged**